src/components/ClassEnrollmentConfirm.tsx

```diff
@@ -0,0 +1,403 @@
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Stack,
+    Flex,
+} from "@chakra-ui/react";
+import { BackButton } from "@components/BackButton";
+import { CloseButton } from "@components/CloseButton";
+import { Student, Parent } from "@prisma/client";
+import colourTheme from "@styles/colours";
+import parsePhoneNumber from "@utils/parsePhoneNumber";
+
+type ClassEnrollmentConfirmationProps = {
+    studentData: Student; // Not actually any but I don't want to type the entire student object, let me know if I have to do this or if it's done somewhere else that can be used here
+    parentData: any; // ^^^ Same idea
+    pageNum: number;
+    setPageNum: React.Dispatch<SetStateAction<number>>;
+};
+
+const therapyMapping = {
+    PHYSIO: "Physiotherapy",
+    SPEECH_LANG: "Speech and language therapy",
+    OCCUPATIONAL: "Occupational therapy",
+    COUNSELLING: "Counseling",
+    ART: "Art therapy",
+};
+
+export const ClassEnrollmentConfirmation = (
+    props: ClassEnrollmentConfirmationProps,
+): JSX.Element => {
+    return (
+        <Box>
+            <Flex justifyContent="space-between" marginBottom="39px">
+                <BackButton
+                    onClick={() => {
+                        props.setPageNum(props.pageNum - 1);
+                    }}
+                />
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/" />
+            </Flex>
+            <VStack spacing="45px" alignItems="flex-start">
+                <HStack
+                    alignSelf="flex-start"
+                    mt="15px"
+                    fontWeight="700"
+                    fontSize="36px"
+                >
+                    <Heading>
+                        Confirm Personal Information{" "}
+                        <Heading color={colourTheme.colors.Blue} as="span">
+                            ({props.studentData.firstName}{" "}
+                            {props.studentData.lastName})
+                        </Heading>
+                    </Heading>
+                </HStack>
+                <Text fontWeight="300" fontSize="22px">
+                    Has any of the following information changed? Please confirm
+                    that none of the following information has changed.
+                </Text>
+                <VStack
+                    alignItems="flex-start"
+                    spacing="32px"
+                    marginBottom="80px"
+                >
+                    <Text fontWeight={700} fontSize="24px" marginBottom="-16px">
+                        General Participant Information
+                    </Text>
+                    <Box>
+                        <Text
+                            color={colourTheme.colors.Gray}
+                            fontSize="14px"
+                            marginBottom="6px"
+                        >
+                            {" "}
+                            Participant Name{" "}
```

Comment:
Are these spaces just for adding margin-left and right? it seems pretty weird not to just do `marginLeft` and `marginRight` in the Text component.

Btw @MatoPlus looks like the codebase could benefit from creating a reusable component called `FormLabel` or something with the color, fontsize,  and margins preset (and overridable) so we don't have to do so much copying and pasting